dev/TitleBar/TitleBar.cpp

```diff
@@ -264,3 +269,16 @@ void TitleBar::UpdateTheme()
         }
     }
 }
+
+void TitleBar::UpdateTitle(const winrt::hstring& title)
```

Comment:
I think you'll want to call this from ApplyTemplate() as well, otherwise it won't be called on load.

Reply:
Done!